engine/src/main/java/com/datatorrent/stram/engine/InputNode.java

```diff
@@ -145,7 +148,7 @@ else if (PROCESSING_MODE == ProcessingMode.EXACTLY_ONCE) {
 
               ContainerStats.OperatorStats stats = new ContainerStats.OperatorStats();
               reportStats(stats, currentWindowId);
-              if(!insideWindow){
+              if(!insideApplicationWindow){
```

Comment:
@PramodSSImmaneni before auto metrics they were. but with auto metrics it was changed and that discussion happened and was discussed.